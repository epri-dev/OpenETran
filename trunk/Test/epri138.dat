* incoming backflash for the EPRI 138-kV training example

time 1 6e-9 10e-6

* network definition

span 1
cable 1 472.0 3.0e8 -93.0e3
end
* end

line 1 2 1 10.67 0 0
line 2 3 1 15.24 0 0
line 2 4 1  9.14 0 0
line 2 5 1  9.14 0 0
line 5 7 1  9.14 0 0
line 7 8 1  9.14 0 0
line 6 7 1 10.67 1 0
line 7 9 1 15.24 0 0
end

labelphase 0 G
labelphase 1 A

labelpole 1 ln1
labelpole 2 cb1
labelpole 3 xf1
labelpole 4 top
labelpole 5 tiebrk
labelpole 6 ln2
labelpole 7 cb2
labelpole 8 bot
labelpole 9 xf2

meter
pairs 1 0
poles all

* source equivalent

resistor 472.0
pairs 1 0
poles 1

steepfront 2.54e3 1.82e-6 9.7e-6 1.0e-7 1.0
pairs 1 0
poles 1

* transformers and arresters

capacitor 2e-9
pairs 1 0
poles 3 9

arrbez 0.0 296.0e3 0.051 1.57e-6 1.83 1
pairs 1 0
poles 3 9

* insulators

lpm 650.0e3 535.0e3 7.785e-7
* lpm 650.0e3 551.3e3 1.831e-6
pairs 1 0
poles all
