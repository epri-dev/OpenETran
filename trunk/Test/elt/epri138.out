  N   span     dT   Tmax
  1   0.00 6e-009 1e-005
Z-phase
  4.72000e+002
Z-modal
  4.72000e+002
Modal Transformation (Ti)
  1.00000e+000
Power-frequency source at pole 6.
   #        vp        vm         i
   1  -93000.0  -93000.0    -197.0
Power-frequency source at pole 1.
   #        vp        vm         i
   1  -93000.0  -93000.0    -197.0
Meter at pole 1, from 1 to 0, max V   = 2.787554e+005
Meter at pole 2, from 1 to 0, max V   = 2.788484e+005
Meter at pole 3, from 1 to 0, max V   = 2.723317e+005
Meter at pole 4, from 1 to 0, max V   = 2.797187e+005
Meter at pole 5, from 1 to 0, max V   = 2.815265e+005
Meter at pole 6, from 1 to 0, max V   = 2.822285e+005
Meter at pole 7, from 1 to 0, max V   = 2.822285e+005
Meter at pole 8, from 1 to 0, max V   = 2.833343e+005
Meter at pole 9, from 1 to 0, max V   = 2.780962e+005
Meter at pole 3, arrester max I       = 6.060712e+002
Meter at pole 9, arrester max I       = 7.784959e+002
insulator at pole 1, from 1 to 0 per-unit SI = 3.461890e-001
insulator at pole 2, from 1 to 0 per-unit SI = 3.496689e-001
insulator at pole 3, from 1 to 0 per-unit SI = 3.519789e-001
insulator at pole 4, from 1 to 0 per-unit SI = 3.497249e-001
insulator at pole 5, from 1 to 0 per-unit SI = 3.511641e-001
insulator at pole 6, from 1 to 0 per-unit SI = 3.524673e-001
insulator at pole 7, from 1 to 0 per-unit SI = 3.527063e-001
insulator at pole 8, from 1 to 0 per-unit SI = 3.527746e-001
insulator at pole 9, from 1 to 0 per-unit SI = 3.552067e-001
nr_iter = 12423, nr_max = 7
Execution time [sec]: 0.062
